python/ray/tune/search/nevergrad/nevergrad_search.py

```diff
@@ -52,6 +52,8 @@ class NevergradSearch(Searcher):
     Parameters:
         optimizer: Optimizer provided
             from Nevergrad. Alter
```

Comment:
```suggestion
        optimizer: Optimizer class provided from Nevergrad.
            See here for available optimizers:
            https://facebookresearch.github.io/nevergrad/optimizers_ref.html#optimizers
            This can also be an instance of a `ConfiguredOptimizer`. See the
            section on configured optimizers in the above link.
```